packages/material-ui-styles/src/withStyles.js

```diff
@@ -83,10 +83,10 @@ const withStyles = (stylesOrCreator, options = {}) => Component => {
         return null;
       }
 
-      return new Error(
-        'Material-UI: The `innerRef` prop is deprecated and will be removed in v5. ' +
-          'Refs are now automatically forwarded to the inner component.',
-      );
+      // return new Error(
```

Comment:
It's raising a warning as the ButtonBase is still using this API. Humm, hold on.

Reply:
Right. I think I annotated it with `@deprecated` somewhere to indicate a "soft deprecation". In a perfect world the eslint rule would detect these. I don't like runtime deprecation warnings that have the same level as any other warning (hello `ReactDOM.findDOMNode`). A lint rule makes more sense to me.